docs/tutorial/3-build-your-first-dapp.md

```diff
@@ -5,9 +5,9 @@ slug: /build-your-first-dapp
 
 # Build Your First Dapp
 
-To get started with building a Decentralized Application (Dapp), clone the following repository and follow the README instructions: [https://github.com/paritytech/contracts-boilerplate](https://github.com/paritytech/contracts-boilerplate).
-
-The repository contains two boilerplates:
-
-1. **React and Ether.js**: This boilerplate uses React and Ether.js, a well-established library for interacting with Solidity contracts.
-2. **Modern TypeScript with wagmi**: This boilerplate offers a more modern approach, fully typed, and includes powerful React hooks provided by wagmi.
+To get started with building a Decentralized Application (Dapp), spin up a repository, using `create-polkadot-dapp`:
+```
+npx create-polkadot-dapp@latest
+```
+Use **react-solidity** template, that contains a monorepo of smart contract repository, together with React front-end application to interact with it, using `ethers` library.  
```

Comment:
Would it be possible to select the template with a CLI arg so that the user does not have to make this choice and possibly select the wrong thing?

Reply:
Combining CLI args with selecting options from prompt would require a bit of engineering, but I was also thinking about it.